base/src/main/java/io/fabric8/launcher/base/http/ExternalRequest.java

```diff
@@ -59,8 +60,8 @@ public static boolean execute(Request request) {
 
     public static <T> Optional<T> executeAndParseJson(Request request, Function<JsonNode, T> jsonNodeFunction) {
         try (Response response = client.newCall(request).execute()) {
+            final ResponseBody body = response.body();
```

Comment:
I think this may fail if response isn't successful, but I may be wrong

Reply:
I took it from the GitHelper and tested it there. Since we never too careful https://github.com/fabric8-launcher/launcher-backend/issues/247